resources/units/groups/Ally-Flak.yaml

```diff
@@ -0,0 +1,14 @@
+name: Ally Flak
```

Comment:
To be 100% correct yes. But i added it there because this makes it more clearer that the Groups will use units from the units defined in this folder. Otherwise we would have to explain what they would have to write into the units part of the group.

It's personal preference i would say. For me its an extension to the already existing UniType class. But actually  i dont really care. If you want it there it can get there.